src/main/java/dev/slimevr/vr/processor/skeleton/HumanSkeleton.java

```diff
@@ -1829,6 +1829,38 @@ public void resetTrackersFull() {
 		this.legTweaks.resetBuffer();
 	}
 
+	private boolean shouldResetMounting(TrackerPosition position) {
+		return position != null
+			// TODO: Feet can't currently be reset using this method, maybe
+			// they'll need a separate step just for them?
+			&& position != TrackerPosition.LEFT_FOOT
+			&& position != TrackerPosition.RIGHT_FOOT;
+	}
+
+	private boolean shouldReverseYaw(TrackerPosition position) {
+		return position != null
+			&& (position == TrackerPosition.LEFT_UPPER_LEG
+				|| position == TrackerPosition.RIGHT_UPPER_LEG
+				|| position == TrackerPosition.LEFT_LOWER_ARM
+				|| position == TrackerPosition.RIGHT_LOWER_ARM
+				|| position == TrackerPosition.LEFT_HAND
+				|| position == TrackerPosition.RIGHT_HAND);
```

Comment:
Those don't need to be reversed, since they'll rotate forward along with the torso, the lower arm and hand rotate backwards if you bend it from that position